### DISCUSSION TITLE
Pot | Lingva

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#749] [General]
# Pot | Lingva

跨平台划词翻译和OCR软件

https://pot-app.com/docs/api/tts/lingva.html

**Comment by Mayeverything** [comment]
语音合成服务提示Network Error: error sending request for url (https://lingva.pot-app.com/api/v1/audio/en/Acknowledgments): error trying to connect: 远程主机强迫关闭了一个现有的连接。 (os error 10054)

**Reply by hero-jianghaojie** [reply]
把lingva删去，使用其他的插件，如

有道TTS[插件]


火山TTS[插件]
